test/integration/verify-test.ts

```diff
@@ -4,63 +4,114 @@ const eventPayload = {
   foo: "bar",
 };
 const secret = "mysecret";
-const signature = "sha1=d03207e4b030cf234e3447bac4d93add4c6643d8";
+const signatureSHA1 = "sha1=d03207e4b030cf234e3447bac4d93add4c6643d8";
+const signatureSHA256 =
+  "sha256=4864d2759938a15468b5df9ade20bf161da9b4f737ea61794142f3484236bda3";
 
-test("verify() without options throws", () => {
-  expect(() => verify()).toThrow();
-});
+describe("throws when calling", () => {
+  test("verify() without options", () => {
+    expect(() => verify()).toThrow();
+  });
 
-test("verify(undefined, eventPayload) without secret throws", () => {
-  expect(() => verify.bind(null, undefined, eventPayload)()).toThrow();
-});
+  test("verify(undefined, eventPayload) without secret", () => {
+    expect(() => verify.bind(null, undefined, eventPayload)()).toThrow();
+  });
 
-test("verify(secret) without eventPayload throws", () => {
-  expect(() => verify.bind(null, secret)()).toThrow();
-});
+  test("verify(secret) without eventPayload", () => {
+    expect(() => verify.bind(null, secret)()).toThrow();
+  });
 
-test("verify(secret, eventPayload) without options.signature throws", () => {
-  expect(() => verify.bind(null, secret, eventPayload)()).toThrow();
+  test("verify(secret, eventPayload) without options.signature", () => {
+    expect(() => verify.bind(null, secret, eventPayload)()).toThrow();
+  });
 });
 
-test("verify(secret, eventPayload, signature) returns true for correct signature", () => {
-  const signatureMatches = verify(secret, eventPayload, signature);
-  expect(signatureMatches).toBe(true);
-});
+describe("for SHA1 signature", () => {
+  test("verify(secret, eventPayload, signature) returns true for correct signature", () => {
```

Comment:
can you mention the different algorithm in the test descriptions so you don't use the same description for two tests? E.g.
```suggestion
  test("verify(secret, eventPayload, signatureSHA1) returns true for correct signature", () => {
```